src/emulate.c

```diff
@@ -30,6 +30,7 @@ extern struct target_ops gdbstub_ops;
 
 #include "riscv.h"
 #include "riscv_private.h"
+#include "utils.c"
```

Comment:
Change `Makefile` and build `utils.o` from `src/utils.c`.

Reply:
Hi @jserv,
Do you have recommendation material for makefile and .o files? 
I think I need to rebuild the basic knowledge for this.

Reply:
Check the official documentation: https://www.gnu.org/software/make/manual/